hazelcast/pom.xml

```diff
@@ -326,14 +326,14 @@
         <dependency>
             <groupId>org.slf4j</groupId>
             <artifactId>slf4j-api</artifactId>
-            <version>1.7.25</version>
+            <version>${slf4j.api.version}</version>
             <scope>provided</scope>
             <optional>true</optional>
         </dependency>
         <dependency>
             <groupId>org.slf4j</groupId>
             <artifactId>slf4j-simple</artifactId>
-            <version>1.5.8</version>
+            <version>${slf4j.api.version}</version>
             <scope>provided</scope>
             <optional>true</optional>
         </dependency>
```

Comment:
Could we remove this completely? It looks suspicious. Why should be one of the implementations required?

Reply:
Better to remove but there is a note from @pveentjer for it at the start of dependencies tag: `<!-- needed for thread affinity -->`. So i kept it as is.

Reply:
@pveentjer Could you comment? Is the `slf4j-simple` implementation really needed?

Reply:
Yes it is needed for the thread affinity library.